zebROS_ws/src/tf_object_detection/src/screen_to_world.cpp

```diff
@@ -140,8 +149,13 @@ int main (int argc, char **argv)
 	image_transport::ImageTransport it(nh);
 
 	std::string algorithm_str;
+	double timeout;
 
 	nh_param.param<std::string>("depth_algorithm", algorithm_str, "CONTOURS");
+	nh_param.param<double>("timeout", timeout, 0.05);
```

Comment:
What is this used for?

Reply:
it's probably not the best named parameter, it's used to set the maximum time delta between messages for the synchronizer. screen to world for the tags wasn't picking up the apriltag detections because they were running at ~6fps compared to the camera's 30, so we had to set that to around 0.2 seconds to get it to work

Reply:
when it was set to 0.05, `/tf_object_detection/tag_detection_world` wasn't publishing anything even though `/tf_object_detection/tag_detection_msg`, `/zed_objdetect/depth/depth_registered`, and `/zed_objdetect/left/camera_info` were all being published

Reply:
I guess my point is that the variable doesn't seem to be referenced in the code?

Reply:
oops... i meant to put it in line 215 in the `ros::Duration` constructor, thanks for noticing that! i'll change it right now

Reply:
ok it should be resolved now, thanks again